modules/gdscript/language_server/lsp.hpp

```diff
@@ -983,32 +983,32 @@ struct CompletionList {
  * A symbol kind.
  */
 namespace SymbolKind {
-static const int File = 1;
-static const int Module = 2;
-static const int Namespace = 3;
-static const int Package = 4;
-static const int Class = 5;
-static const int Method = 6;
-static const int Property = 7;
-static const int Field = 8;
-static const int Constructor = 9;
-static const int Enum = 10;
-static const int Interface = 11;
-static const int Function = 12;
-static const int Variable = 13;
-static const int Constant = 14;
-static const int String = 15;
-static const int Number = 16;
-static const int Boolean = 17;
-static const int Array = 18;
-static const int Object = 19;
-static const int Key = 20;
-static const int Null = 21;
-static const int EnumMember = 22;
-static const int Struct = 23;
-static const int Event = 24;
-static const int Operator = 25;
-static const int TypeParameter = 26;
+static const int File = 0;
```

Comment:
Why use constants in a namespace instead of an enum?

Reply:
As identifiers like `String`, `Array` are already used in godot.

Reply:
But that shouldn't matter if you give the enum a name?
```
enum SymbolKind {
	FILE,
	MODULE,
	NAMESPACE,
...
};
```
(enum values should be uppercase)

Reply:
I'd like to use `lsp::SymbolKind::File` instead of `lsp::FILE` to keep the same name with the [LSP specifications](https://microsoft.github.io/language-server-protocol/specifications/specification-3-14/#textDocument_documentSymbol)

Reply:
Ok, then you can keep them PascalCase, but I think you can still use an enum and call the values with `lsp::EnumName::EnumValue` a priori.

Reply:
I tried that. The compiler compains about the `String`, `Array` etc.

Reply:
Ah I see. I guess it's fine as is then.

Reply:
You can use `enum class` if you don't want to allow `lsp::File`.

Reply:
It should have a comment explaining that then.